### PR TITLE
[chore] Add connecting to Strapi section to CONTRIBUTING

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

Adds a section describing how the design system can be connected to Strapi for testing in the CMS

### Why is it needed?

Testing in isolation using Storybook is great but you might also want to test your work in the CMS

### How to test it?

Make a change somewhere in the design system, then follow the steps to link the design system to your Strapi application. Verify the local version of the DS is being used.

--- 

More or less the same as https://github.com/joshuaellis/strapi-design-system#connecting-to-strapi

### CODE REVIEW

**Review by joshuaellis — APPROVED** [review]
Lovely.

File: README.md [hunk 1]
```diff
@@ -66,3 +66,29 @@ If you encounter problems doing so, run the following command and retry the prev
...
+
+Replace the version number in both `strapi/packages/core/admin/package.json` and `strapi/packages/core/helper-plugin/package.json` with the relative path to your copy of the design system:
+
+```
+"@strapi/design-system": "link:../../../../strapi-design-system/packages/strapi-design-system"
+```
+
+### Yarn link
+
+Alternatively, you can use [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/) by first running `yarn link` in `strapi-design-system/packages/design-system` and then `yarn link @strapi/design-system` in both `strapi/packages/core/admin` and `strapi/packages/core/helper-plugin`. With this approach no changes need to be made to the `package.json`
```
Comment (remidej) [inline README.md:88]:
```suggestion
Alternatively, you can use [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/) by first running `yarn link` in `strapi-design-system/packages/design-system` and then `yarn link @strapi/design-system` in both `strapi/packages/core/admin` and `strapi/packages/core/helper-plugin`. With this approach, no changes need to be made to the `package.json`
```

File: README.md [hunk 2]
```diff
@@ -66,3 +66,29 @@ If you encounter problems doing so, run the following command and retry the prev
 ```sh
 $ git reset --hard
 ```
+
+## Connecting to Strapi
+
+If you want to analyze the admin bundle and the impact each library has on the over all application you can link this repo to your copy of [@strapi/strapi](https://github.com/strapi/strapi).
```
Comment (remidej) [inline README.md:72]:
```suggestion
If you want to analyze the admin bundle and the impact each library has on the overall application, you can link this repo to your copy of [@strapi/strapi](https://github.com/strapi/strapi).
```

File: README.md [hunk 3]
```diff
@@ -66,3 +66,29 @@ If you encounter problems doing so, run the following command and retry the prev
...
+
+First, run `yarn build` in `strapi-design-system/packages/strapi-design-system` to generate the bundle.
+
+In your copy of Strapi you can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).
```
Comment (remidej) [inline README.md:76]:
```suggestion
In your copy of Strapi, you can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).
```

**Review by remidej — COMMENTED** [review]
👌 (just added some comas)

File: README.md [hunk 4]
```diff
@@ -66,3 +66,29 @@ If you encounter problems doing so, run the following command and retry the prev
 ```sh
 $ git reset --hard
 ```
+
+## Connecting to Strapi
```
Comment (gu-stav) [inline README.md:70]: Wdyt about 
```suggestion
## Linking the design-system in your application
```

**Review by gu-stav — COMMENTED** [review]
It wouldn't make this about the monorepo but more about how to link it in general, otherwise I'd prefer to move this into the contributor docs of the monorepo. What do you think?

**Comment by markkaylor** [comment]
> It wouldn't make this about the monorepo but more about how to link it in general, otherwise I'd prefer to move this into the contributor docs of the monorepo (because the DS is supposed to be independent from the monorepo). What do you think?

That's a good point. I think making it more general to Strapi is a good idea but not so general that people think they can (or should) use the DS in any React application. What if I adapted what I have here to have instructions for both the monorepo and Strapi applications/plugins?

**Comment by gu-stav** [comment]
Yeah I think that could work. If you split it up anyways I think I'd prefer to have the strapi specific part in the contributor docs and the general part in the DS.

**Review by gu-stav — APPROVED** [review]

**Comment by markkaylor** [comment]
Hey @gu-stav thanks for the approval, but I had worked on your feedback yesterday and did the contributor docs yesterday. Let me know what you think. This PR https://github.com/strapi/strapi/pull/15439 will need to be merged first and I guess we will have to wait for the release and website deploy since I link to it here.

**Comment by gu-stav** [comment]
@markkaylor Are you going to refactor this still (take out the Strapi specific part and link to the contributor docs instead + write more generalized docs)?

**Comment by markkaylor** [comment]
@gu-stav I did link to the contributor docs for the monorepo here and then left more general instructions for connecting to a Strapi application or plugin. I thought that was the idea but I guess I misunderstood [your comment here](https://github.com/strapi/design-system/pull/836#issuecomment-1377330848). I can make it more general to any React application but I do feel the strapi-design-system is quite specific to Strapi

**Comment by markkaylor** [comment]
Hi @gu-stav have you had a chance to take another look? If the content looks good to you I was thinking the CONTRIBUTING.md might be a better place for this.

File: README.md [hunk 5]
```diff
@@ -66,3 +66,31 @@ If you encounter problems doing so, run the following command and retry the prev
...
+First, run `yarn build` in `strapi-design-system/packages/strapi-design-system` to generate the bundle.
+
+You can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).
+
+### Relative path
+
+In the package.json of your React application replace the version number with the relative path to your copy of the design system:
+
+```
+"@strapi/design-system": "link:../../../../strapi-design-system/packages/strapi-design-system"
```
Comment (gu-stav) [inline README.md:87]:
```suggestion
"@strapi/design-system": "link: <relative-path>/strapi-design-system/packages/strapi-design-system"
```

**Review by gu-stav — COMMENTED** [review]
There is only one small thing I'd change, but moving it to the contrib markdown-file sounds like an excellent idea!

**Review by gu-stav — APPROVED** [review]
Very cool, thanks!

**Pull request merged**